app/controller/sdl/Abstract/Controller.js

```diff
@@ -1632,6 +1636,18 @@ SDL.SDLController = Em.Object.extend(
           frames[i].style.pointerEvents = "none";
         }
       }
-    }
+    },
+
+    showWebViewApp: function(appID) {
+      for(var key in SDL.SDLModel.webApplicationFramesMap) {
+        SDL.SDLModel.webApplicationFramesMap[key].hidden = key != SDL.SDLModel.appIDtoPolicyAppIDMapping[appID];
```

Comment:
@YarikMamykin I believe you can reuse function below:
```
this.hideWebApps();
SDL.SDLModel.webApplicationFramesMap[appID].hidden = true;
```

BTW maybe you can pass policy app ID to that function from outside? In that case you won't need an intermediate map

Reply:
@AKalinich-Luxoft unfortunately, I see no way to pass policyAppID to this function, so the mapping is still needed. Also I think, as soon as it captures all applications this mapping should be useful in future. 
Please take a look at https://github.com/LuxoftSDL/sdl_hmi/pull/39/commits/a725b2ecd972b868047011818fc0e1b26fb2b29a